notebooks/ndd_example.ipynb

```diff
@@ -0,0 +1,202 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ed9dd0a2-6b12-436f-b6c3-02e0072115e3",
+   "metadata": {},
+   "source": [
+    "This notebook shows the results of query-time near-duplicate detection (NDD).  It shows the same query with and without the duplicates.  To use this notebook, first start the Sycamore containers using `docker compose up`.  Then, ingest the college credit card marketing agreement database.  There are two choices here.  The quickest approach is to ingest only the 35 documents that are needed for the query below.  This can be accomplished by running `docker compose run sycamore_crawler_s3 -anon aryn-public cccmad-tiny`.  The other option is to ingest all 1911 PDFs that have text, by running `docker compose run sycamore_crawler_s3 -anon aryn-public cccmad`.  This will take hours and around 20 PDFs will have problems.  Once ingestion has sufficiently completed, run this notebook."
```

Comment:
"Taking hours and 20 PDFs having problems" - what problems are there? Does it fail the job, or just not show up in query results. Also, how long did it take on your MacBook (or whatever we tested)? Do we know why it takes so long to do this?

Reply:
Seems like 4-6 hours on my laptop (Linux, 16 cores, 32GB).  Of 1911 PDFs, 17 failed to ingest for some reason.  The importer keeps trying, but the failures seem persistent.

Reply:
Gotcha. Do we know how long the retrying period lasts? E.g. is it 4 hours of processing, 20 mins of flailing?

Reply:
The retrying continues forever, I think.

Reply:
That seems like an issue. @eric-anderson we don't gracefully fail?